README.md

```diff
@@ -14,6 +14,7 @@ with files, event file descriptors, ioctls and others.
 
 **Operating Systems**:
 - Linux
+- Windows (partial support)
```

Comment:
We also need to update this phrase: "The crate implements safe wrappers around common Linux utilities for working
with files, event file descriptors, ioctls and others."

Reply:
Done